apps/curbside_concerts_web/lib/curbside_concerts_web/views/musician_view.ex

```diff
@@ -2,4 +2,56 @@ defmodule CurbsideConcertsWeb.MusicianView do
   use CurbsideConcertsWeb, :view
 
   alias CurbsideConcerts.Musicians.Musician
+
+  def required_star do
+    ~E|<span class="required">*</span>|
+  end
+
+  def musician_input(form, field, placeholder \\ "") do
+    class = class(form, field)
+
+    ~E"""
+    <%= text_input(form, field, class: class, placeholder: placeholder) %>
+    """
+  end
+
+  def musician_inline_input(form, field, placeholder \\ "") do
+    class = "inline-input " <> class(form, field)
+
+    ~E"""
+    <%= text_input(form, field, class: class, placeholder: placeholder) %>
+    """
+  end
+
+  def musician_input(form, field, placeholder, type: :url) do
+    class = class(form, field)
+
+    ~E"""
+    <%= url_input(form, field, class: class, placeholder: placeholder) %>
+    """
+  end
+
+  def musician_textarea(form, field, placeholder \\ "") do
+    class = class(form, field)
+
+    ~E"""
+    <%= textarea(form, field, class: class, placeholder: placeholder) %>
+    """
+  end
+
+
+  def musician_select(form, field) do
+    class = class(form, field)
+
+    options = ["Voluneteer", "Paid"]
```

Comment:
spelling: Volunteer

From what I've seen, sometimes the same musician does some Paid and some Volunteer work. Was this a field that they asked for?